packages/fluentui/react-northstar/src/themes/teams/components/Chat/chatMessageContentStyles.ts

```diff
@@ -0,0 +1,55 @@
+import { ComponentSlotStyleFunction, ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+
+import { ChatMessageContentStylesProps } from '../../../../components/Chat/ChatMessageContent';
+import { ChatMessageVariables } from './chatMessageVariables';
+import { pxToRem } from '../../../../utils';
+
+const getChatMessageVariantStyles: ComponentSlotStyleFunction<ChatMessageContentStylesProps, ChatMessageVariables> = ({
+  props: p,
+  variables: v,
+  theme,
+}) => {
+  return {
+    ...(p.layout === 'refresh' &&
+      p.density === 'comfy' && {
```

Comment:
Why does the new link design only apply to comfy density?
Shouldn't it be consistent for all refresh designs? (@davezuko)